Autopilot/Inc/batterysensor.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Battery Sensor Header File
+ * 
+ * This will basically sense the voltage of the battery.
+ * 
+ * Author: Sahil Kale
+ *  
+ **/
+
+#ifndef BATTERYSENSOR_HPP
+#define BATTERYSENSOR_HPP
+
+/**
+ * Starts the ADC for the battery sensor 
+ */
+virtual void initBatterySensor();
+
+/**
+ *  Returns the battery voltage as a float
+ * 
+ * @param battery The selected battery to return the voltage from
+ */
+virtual float getBatteryVoltage(int battery);
+
+#endif
```

Comment:
make sure to always add 1 blank line at the end of your files. They're currently both missing one. Not a huge deal, but will produce warnings every once in a while. I recommend you just rig your text editor to automatically add a blank line on save.

Reply:
Sounds good!